pgmoon/init.moon

```diff
@@ -2,8 +2,13 @@ socket = require "pgmoon.socket"
 import insert from table
 
 import rshift, lshift, band from require "pgmoon.bit"
-pl_file = require "pl.file"
-ssl = require "ngx.ssl"
+
+local pl_file
+local ssl
+
+if ngx
+  pl_file = require "pl.file"
+  ssl = require "ngx.ssl"
 
 unpack = table.unpack or unpack
 
```

Comment:
is moonscript code necessary for us (kong) since we always run in openresty with luajit?

Reply:
I think the Lua code are compiled from `.moon` script, so it is best to not edit the `.lua` file directly, as any changes made may be lost.